### DISCUSSION TITLE
Running misspelled alias commands

### DISCUSSION THREAD

**Discussion by dingo-d** [wp-cli/wp-cli#5813] [Ideas]
This is an idea coming from this issue: https://github.com/wp-cli/extension-command/issues/248

So, in essence, when we mistype a command, we would get a recommendation like:

> Did you mean 'delete'?

It would be good to have an option (Symfony commands have this options) to actually run the suggested option, instead of just exiting out just to type the command again.

>$ wp plugin remove plugin-name
> Did you mean 'delete'? [Y]

when you type Y, yes or y, you'd run the `wp plugin delete plugin-name` command.

**Comment by swissspidy** [comment]
Created an issue for this feature request: https://github.com/wp-cli/wp-cli/issues/6054
